src/moduler/verktoylinje/verktoylinje.js

```diff
@@ -6,34 +6,18 @@ import Filter from '../filtrering/filter';
 import PeriodeFilter from '../filtrering/filter/periode-filter';
 import { selectViserHistoriskPeriode } from '../filtrering/filter/filter-selector';
 import { selectErUnderOppfolging, selectHarSkriveTilgang } from '../oppfolging-status/oppfolging-selector';
-import { selectDialoger, selectHarTilgangTilDialog } from '../dialog/dialog-selector';
-import { dialogFilter } from '../filtrering/filter/filter-utils';
 import InternLenke from '../../felles-komponenter/utils/internLenke';
 import VisValgtFilter from '../filtrering/filter-vis-label';
 import { selectHarTilgangTilAktiviteter } from '../aktivitet/aktivitet-selector';
-import { hentDialog } from '../dialog/dialog-reducer';
 import loggEvent, { APNE_NY_AKTIVITET, APNE_OM_TJENESTEN } from '../../felles-komponenter/utils/logging';
-import { DialogLenkeToggel } from './DialogLenkeToggel';
 
 class Verktoylinje extends Component {
-    componentDidMount() {
-        const { doHentDialog } = this.props;
-        doHentDialog();
-    }
-
     render() {
-        const {
-            viserHistoriskPeriode,
-            underOppfolging,
-            harSkriveTilgang,
-            antallUlesteDialoger,
-            aktivitetLaster,
-            dialogLaster
-        } = this.props;
+        const { viserHistoriskPeriode, underOppfolging, harSkriveTilgang, aktivitetLaster } = this.props;
         return (
             <div className="verktoylinje">
                 <div className="verktoylinje__verktoy-container">
-                    <DialogLenkeToggel dialogLaster={dialogLaster} antallUlesteDialoger={antallUlesteDialoger} />
+                    <div />
```

Comment:
Trengs det en tom div her ?

Reply:
Hvis du ser i `verktoylinje.less` så ser du utstrakt bruk av `nth-child`, noe jeg syntes var såpass uryddig at jeg oppretta [FO-2836](https://jira.adeo.no/browse/FO-2836).

For å komme videre i livet la jeg inn en `<div/>`, reverterte `verktoylinje.less` og oppretta en issue på å rydde opp.